### ISSUE TITLE
multi label/tag classification

### ISSUE THREAD

**Issue by prabirshrestha** [guillaume-be/rust-bert#40]
Is there an example to generate tags/labels on given sentences?

Looking at something similar to fasttext multi label classification.
https://fasttext.cc/docs/en/supervised-tutorial.html#multi-label-classification

I would also like to see examples on how to train them.

**Comment by guillaume-be** [comment]
Hello,

The current classification pipeline only supports multi-class sequence classification. I can push an update to support multi-label with threshold as a ready-to-use pipeline. I do not have a public pre-trained model that would work as an example, so you would have to bring in your own model.

The training of the models in this library is possible, but no example is available yet. If you do not want to write a training loop in Rust, the easiest may be to use existing Python scripts to train these models and them load them into Rust.

**Comment by guillaume-be** [comment]
Implementation of multi-label sequence classification to be merged following checks: https://github.com/guillaume-be/rust-bert/pull/41

**Comment by guillaume-be** [comment]
Multi-label classification has been implemented in https://github.com/guillaume-be/rust-bert/pull/41. Closing this issue for now - please don't hesitate to reach out if this does not address your requirements.
